### PR TITLE
General info #2

### PR DESCRIPTION
**Pull request by MechanicKim**
I fixed
1. EditForm.js
- style errors
- TextInput underline problem in Android

2. GeneralInfo.js
- Added Location field.

### CODE REVIEW

**Comment by lyw0149** [comment]
Now we can upload NICE profile picture!!!!!
Thank you minkyuminkyu!!!! LGTM

File: src/components/SignUp/EditForm.js [hunk 1]
```diff
@@ -26,49 +41,40 @@ class EditForm extends Component {
     });
   }
 
-  render() {
-    return this.renderEdit();
+  // When you finish editing, this component is changed into none edit mode.
+  reflctInput() {
```
Comment (SarahhhC) [inline src/components/SignUp/EditForm.js]: reflct => reflect please

**Comment by SarahhhC** [comment]
Approved! LGTM 💩

**Comment by ace0625** [comment]
LGTM

**Comment by SarahhhC** [comment]
LGTM 👍 

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/97/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**